js/localstorage.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+/* eslint-disable func-style */
+const userFullName = document.getElementById('full-name');
+const userLastName = document.getElementById('Last-name');
+const userEmail = document.getElementById('email-address');
+const userMessage = document.getElementById('message');
+
+function setLocal () {
+  const localName = localStorage.getItem('full-name');
+  const localLastName = localStorage.getItem('Last-name');
+  const localemail = localStorage.getItem('email-address');
+  const localText = localStorage.getItem('message');
+  document.getElementById('full-name').value = localName;
+  document.getElementById('Last-name').value = localLastName;
+  document.getElementById('email-address').value = localemail;
+  document.getElementById('message').value = localText;
+}
+
+function populateForm () {
+  localStorage.setItem('full-name', document.getElementById('full-name').value);
+  localStorage.setItem('Last-name', document.getElementById('Last-name').value);
+  localStorage.setItem('email-address', document.getElementById('email-address').value);
+  localStorage.setItem('message', document.getElementById('message').value);
+  setLocal();
+}
+
+if (localStorage.getItem('full-name')) {
+  populateForm();
+} else {
+ setLocal();
+}
```

Comment:
**Hello your localStorage is awesome, but let's make it great**

- [ ] Your **`localStorage`** is storing every input field in a single object, but it should be one object to store all the fields  [see requirements](https://user-images.githubusercontent.com/61562626/128422302-91d3d9ef-336d-4f52-963b-5c69892ba91f.jpeg)

- [ ]  The data retrieved from the **`localStorage`** is vanishing after reloading the page ![status](https://user-images.githubusercontent.com/61562626/128422619-5c636ee5-1fec-4c9d-8ecd-aef05bc95298.jpeg)

Reply:
Issue resolved ✔️